config/rapidez/system.php

```diff
@@ -22,6 +22,9 @@
     // With this token you can run commands from an url.
     'admin_token' => env('RAPIDEZ_TOKEN', env('APP_KEY')),
 
+    // Whether or not to index products with visibility 1 into ElasticSearch
+    'index_invisible' => false,
```

Comment:
If we want the setting to differ for other store views that's not possible.
I suggest wrapping the config getter in [value()](https://laravel.com/docs/10.x/helpers#method-value)

This way a closure could be used as well, allowing for customisation of the return value

Reply:
I don't think this setting will be different per store

Reply:
Might still be worth having the option though, I can imagine some crazy multistore setup could make use of it

(but then maybe we could think about making everything like that and that might be very overkill 🤔 )